### PR TITLE
Export CWE catalogue

### LINKED ISSUE #21: Add script to create cwec.js
As the CWE list gets updated now and then, we should add a script to generate the `cwec.js` file.

### PR DESCRIPTION
**Pull request by domachine**
This PR updates and exposes the cwe catalogue from https://cwe.mitre.org/data/downloads.html.
Closes #21 .

**HEADS UP**: This PR bases on #9 and seems to be larger than it actually is.

### CODE REVIEW

File: tests/cwe.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { expect } from 'chai'
+import * as cwe from '../cwe.js'
+import cweCatalogue from '../lib/shared/cwec.js'
+
+describe('cwe', function () {
+  it('exports all cwe weaknesses', function () {
+    expect(cwe.weaknesses).to.deep.equal(cweCatalogue.weaknesses)
```
Comment (valentindey) [inline tests/cwe.js:7]: this test feels somewhat redundant as both, `cwe` and `cweCatalogue` are imported from the same file :thinking:

**Comment by valentindey** [comment]
I only checked the changes added on top of #9 , these LGTM.

**Review by valentindey — APPROVED** [review]

File: tests/cwe.js (hunk 1 above)
Comment (domachine) [reply tests/cwe.js:7]: Yes, but I wanted to express that the interface is exported and simulate importing like a consumer of the lib would do.

Comment (valentindey) [reply tests/cwe.js:7]: I'd still argue about the significance of this test but am also fine with leaving it there.

**Review by valentindey — APPROVED** [review]

File: tests/cwe.js (hunk 1 above)
Comment (domachine) [reply tests/cwe.js:7]: So, let's argue 😛 ... just kiddin'. If you want, I can remove it, but I feel better to have it there, especially when extending this and having a spot to continue.

File: scripts/cwe-importCatalogue.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+import { writeFile, readFile } from 'node:fs/promises'
+import prettier from 'prettier'
+import xml2js from 'xml2js'
+
+const [, , REGISTRY_FILE] = process.argv
```
Comment (tschmidtb51) [inline scripts/cwe-importCatalogue.js:7]: Maybe add a link here or in the Readme where to download the [latest CWE catalogue](https://cwe.mitre.org/data/xml/cwec_latest.xml.zip)...

**Review by tschmidtb51 — DISMISSED** [review]
Just a minor comment to add a link :-) Other than that: LGTM.

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
Please regenerate the cwec.js again - there is already a new version (4.8) available :see_no_evil:

**Comment by domachine** [comment]
> Please regenerate the cwec.js again - there is already a new version (4.8) available 🙈

The script definitely pays off 😅

**Pull request merged**